ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/KsqlRequest.java

```diff
@@ -70,6 +71,14 @@ public String getKsql() {
     return ksql;
   }
 
+  public String getMaskedKsql() {
+    return maskedKsql;
```

Comment:
Are we ever worried about this being uninitialized? Ideally we'd populate the field in the constructor (see comment above). If for some reason we can't do that, I think we should either default this to the unmasked version or throw if this is uninitialized -- maybe default in `toString()` and throw here -- rather than printing `null` in error messages.

Reply:
Sure. We can throw exception here.